Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Io_image_plugin.cpp

```diff
@@ -222,7 +222,7 @@ class Io_image_plugin :
       connect(planeSwitch, SIGNAL(triggered()),
               this, SLOT(selectPlanes()));
       Scene* true_scene = dynamic_cast<Scene*>(scene);
-      connect(true_scene,SIGNAL(selectionChanged(int)),
+      connect(true_scene,SIGNAL(itemIndexSelected(int)),
```

Comment:
`Scene::itemIndexSelected(int)` is a new signal added by this PR. Why is it useful, compared to `selectionChanged(int)`?

Add, actually, in `Scene`, why are there two functions that seem to do the same?
  - `void setSelectedItemIndex(int i)`
  - `void setSelectedItem(int i )`